dashboard/legacy/middleware/channels_api.rb

```diff
@@ -334,9 +334,17 @@ class ChannelsApi < Sinatra::Base
     # from students so we increase the abuse score enough to block the project
     # with only one report from a verified teacher.
     #
-    # Temporarily ignore anonymous reports and only allow verified teachers to report.
-    # TODO: come up with a better long-term solution here.
-    amount = verified_teacher? ? 20 : 0
+    # Temporarily ignore anonymous reports and only allow verified teachers
+    # and signed in users to report.
+    restrict_reporting_to_verified_users = DCDO.get('restrict-abuse-reporting-to-verified', true)
```

Comment:
Worth mentioning that I defaulted this restriction to true. Happy to change that or just flip the flag once this hits prod.

Reply:
seems fine -- no need to have an additional step after this ships, we just want the ability to turn it off quickly thereafter.